### ISSUE TITLE
Add CI to check `.zenodo.json` formatting 

### ISSUE THREAD

**Issue by freemansw1** [tobac-project/tobac#178]
The `.zenodo.json` file that we are implementing in #139 can be finicky on the Zenodo side. We should add a CI to automatically validate this file.  Zenodo does not(!) release schema for this upload, beyond something very basic required for all Zenodo uploads. This group has made an unofficial schema: https://github.com/zenodraft/metadata-schema-zenodo which seems to work well. The latest version is here: https://zenodraft.github.io/metadata-schema-zenodo/latest/schema.json 
 
This should be reasonably easy to implement through GitHub Actions or Docker or something. It'd be nice to get this in sometime after #139 is merged into `RC_v1.4.0`.

**Comment by freemansw1** [comment]
#292 has been merged, so I will close this.
